src/Calculator/Views/CalculatorProgrammerRadixOperators.xaml.cpp

```diff
@@ -38,18 +38,14 @@ void CalculatorProgrammerRadixOperators::LoadResourceStrings()
     m_logicalShiftButtonContent = resProvider->GetResourceString(L"logicalShiftButtonSelected");
     m_rotateCircularButtonContent = resProvider->GetResourceString(L"rotateCircularButtonSelected");
     m_rotateCarryShiftButtonContent = resProvider->GetResourceString(L"rotateCarryShiftButtonSelected");
+    m_selectedShiftButtonContent = m_arithmeticShiftButtonContent;
```

Comment:
Lets move this out of this function since this isn't a resource string

Reply:
I'll roll back the check Default BitShift function and modify its implementation.